march_gait_files/airgait-v/home/home.gait

```diff
@@ -1,5 +1,9 @@
 name: home
-
-graph:
-  from_subgait: [start   , home_sit, start     , home_stand]
-  to_subgait:   [home_sit, end     , home_stand, end       ]
\ No newline at end of file
+subgaits:
+  home_sit:
+    to: end
+  home_stand:
+    to: end
+  start:
+    stop: home_stand
+    to: home_sit
```

Comment:
I think we should make the home gaits two separate gaits, because this does not make much sense.

Reply:
At first I thought I couldn't separate them, because that would change the name, but now that I think about it that does not matter. I will make the necessary changes in this PR and https://github.com/project-march/march/pull/552